docs/integrations/sigrid-api-documentation.md

```diff
@@ -468,6 +468,244 @@ This end point will return the following response structure:
       "OSH_MAX_SEVERITY": "LOW",
       "TEST_CODE_RATIO": 0.8
     }
+    
+## Managing user permissions via API
+
+In addition to the general usage of the Sigrid API, users also can also perform user management tasks via the API as an alternative to doing these tasks within the web-based user interface of Sigrid itself. This also allows Sigrid administrators to better construct automated processes for managing the access to systems for their users.
+
+### Usage
+
+* The Sigrid UM API base URL is `https://sigrid-says.com/rest/auth/api`.
+* Authentication for the Sigrid API uses the same [authentication tokens](../organization-integration/authentication-tokens.md) that are used by Sigrid CI.
+* You need to pass the authentication token to each request in the HTTP header: `Authorization: Bearer {SIGRID_PERSONAL_TOKEN}`.
+* All end points will return HTTP status 401 if the token is invalid, expired or revoked.
+* All end points will return HTTP status 403 if the token belongs to an non-admin user.
+* All end points return JSON and therefore return a Content-Type of `application/json`.
+
+The following example shows how to call the User Management API via `curl`:
+
+```
+curl -H 'Authorization: Bearer {SIGRID_PERSONAL_TOKEN}' https://sigrid-says.com/rest/auth/api/user-management/{customer}/users
+```
+
+### Managing individual user permissions
+
+A number of endpoints are available that make managing existing users within the portfolio easier:
+
+- `GET https://sigrid-says.com/rest/auth/api/user-management/{customer}/users`: Returns a ist all users within a portfolio
+- `GET https://sigrid-says.com/rest/auth/api/user-management/{customer}/users/{userId}`: Returns a user based on their unique identifier
```

Comment:
lower-case letter after colon?

Reply:
You mean "Returns" into "returns"? In American writing it is typical to capitalize after the colon, in British english this may be different but I was told we're targeting US style writing for our documentation.

Reply:
And I think we should end with a point.